res/mkn/clang.yaml

```
@@ -0,0 +1,21 @@
+property:
+  cxx.flags: --std=c++20 -fPIC -fsized-deallocation
+  lnk.flags: -Wl,-z,defs -fuse-ld=gold 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)